client/app.jsx

```diff
@@ -25,7 +25,7 @@ class App extends React.Component {
 
     var listingId = this.state.listingId
 
-    axios.get(`/api/reservation/${listingId}`)
+    axios.get(`http://localhost:5000/api/reservation/${listingId}`)
```

Comment:
Ooh exactly what we were JUST talking about

Reply:
lol yuuuup now it should work fine